encryption_works.md

```diff
@@ -120,103 +120,100 @@ When Snowden [was answering questions on Guardian's website](http://www.guardian
 
-### Sessions
-
-If you want to use OTR to talk privately with your friends, your friends also need to be using it. An encrypted session between two people requires two encryption keys. For example, if you and your friend are both logged into Facebook chat using Adium or Pidgin and you have both configured OTR, you can chat in private. However if you are logged into IM using Adium or Pidgin but your friend is chatting directly from facebook.com in a web browser, you cannot have an encrypted conversation.
+When you start a new OTR session, your OTR software and your friend's OTR software send a series of messages back and forth to agree upon a new session key. This temporary encryption key, which is only known by your IM clients and is never sent over the Internet, is then used to encrypt and decrypt messages. When the session is finished both clients forget the key. If you start chatting with the same person later, your clients generate a brand new session key.
 
-If you wish to use Facebook or Google's services to chat with your friends, I recommend disabling chat within the web interface of these services and only using Adium and Pidgin to connect, and encouraging all of your friends to do the same thing. Here is instructions on how to do so for [Facebook](https://www.facebook.com/help/215888465102253/) and [Google](https://support.google.com/chat/bin/answer.py?hl=en&answer=161823).
+In this way, even if an eavesdropper is logging all of your encrypted OTR conversations -- which NSA believes it is legally [allowed to do](http://www.forbes.com/sites/andygreenberg/2013/06/20/leaked-nsa-doc-says-it-can-collect-and-keep-your-encrypted-data-as-long-as-it-takes-to-crack-it/) in many cases, even if you're a US citizen and they don't have a warrant or probable cause -- and later they compromise your OTR key, they cannot use it to go back and decrypt your old conversations.
 
-When you start an encrypted OTR session, your client software will tell you something like this:
+This property is called forward secrecy, and it is a feature that OTR has which PGP does not. If your PGP secret key (more on this below) gets compromised, and the attacker has access to all the encrypted messages you've received, they can go back and decrypt them all.
 
-    Attempting to start a private conversation with username@jabberservice...
-    Unverified conversation with username@jabberservice/ChatClient started.
+### A Note About Gmail's "off the record" function.
 
-If you have already verified the OTR fingerprint of the person you're talking with (more on this below) your session will look like this:
+When using Google Hangouts for instant messaging, there's a setting to disable "Hangout history," and it's useful to differentiate between this feature and what the software Off-The-Record offers.
```

Comment:
This section needs work. You should lead in with mentioning the "off the record" feature in Google, since the purpose of this section is to clarify the naming collision. It doesn't make sense to start talking about "hangout history." But you should still mention that you may or may not see this feature depending on which version of Google's chat you are using (Hangouts or "classic").

There used to be a screenshot here. We should keep it, it's very helpful to explain what we are talking about and where it is found in the Gmail UI.